content/en/tracing/trace_collection/runtime_config/_index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Configuration at Runtime
+kind: documentation
+further_reading:
+- link: "/agent/remote_config/"
+  tag: "Documentation"
+  text: "Remote Configuration"
+---
+
+## Overview
+
+Configuration at runtime lets you modify APM library configuration from the Datadog UI, without needing to restart your application or service. You don't need to wait for a new deployment or code change to update your configuration. Instead, update it right away with configuration at runtime.
+
+This feature uses [Remote Configuration][1] to apply configurations to Datadog components in your environment.
+
+{{< img src="/tracing/runtime_config/runtime-config-nav.mp4" alt="Walk through Service Catalog to use configuration at runtime." video="true" style="width:100%;">}}
+
+## Prerequisites
+
+- [Datadog Agent][2] 7.41.1 or higher
+- [Remote Configuration][1] is enabled for your Agent
+- [Read and write permission][4] to APM.
+  **Note**: If you don't have these permissions, ask your Datadog Admin to update your permissions from your Organization Settings.
+
+## Using configuration at runtime
+
+To make changes to a services configuration at runtime:
+
+1. Go to the Service Catalog in APM.
+1. Hover on the service for which you want to update configuration.
+1. Click **Full Page** next to the service name.
+1. Click **Service Info**.
+1. From the **Setup Guidance** tab, click **Edit**.
+1. Change the configuration options as needed. See [supported configuration options](#supported-configuration-options) for more details.
+1. Click **Apply Configuration**.
+
+**Note**: You can also access configuration at runtime from the Trace View panel. Make sure you select an environment for your service to see it.
+
+In **Active Library Configuration**, you can see which options are configured for this service and selected environment:
+
+{{< img src="/tracing/runtime_config/active-library-config.png" alt="From the Setup Guidance tab, you can see your active library configuration." style="width:100%;">}}
+
+In this example, you can see that Log Injection is enabled for the Staging environment across two instances. An instance refers to an instance of the Remote Configuration client. There should be on instance per process of your application.
+
+You can tell when the configuration changes have been successfully applied by referencing the **X Applied** text. In this example, the configuration applied successfully to all two instances.
+
+## Supported configuration options
+
+The following options are supported with configuration at runtime. The required tracer version is listed for each language: 
+
+| Option                                           | Java | Javascript | Python | .NET | Ruby | Go | PHP | C++ |
```

Comment:
I would refer to the table here for the most up to date availability data
https://datadoghq.atlassian.net/wiki/spaces/TS/pages/3119775797/Troubleshooting+APM+Library+Configuration+via+the+UI+An+Overview#Current-Status

Note this table does highlight the Python availability version, however at this time the dd-trace-py v`2.5.0` has NOT been released.

Reply:
Thanks! @avivenzio-dd , I may have misunderstood the table initially. It appears that the Python availability version is only listed for Custom Span Tags. Does this apply to the other options as well?

Also, when more than one version is listed (like with Java), was I correct in interpreting that these different versions apply individually to each of the three options?

Reply:
Let me take a stab at reformatting the table to be more like the one you built in the docs. I can see how it can be misleading. 

Python `2.5.0` when released will support all 4 options and will be the first version to have any support.
If more than one version is listed it will apply to all of the options in that row. Some languages, like js, add new features to multiple major versions, which is why there may be multiple versions in the same row.

custom tags is currently supported for Go, .net, js, & python (when 2.5.0 goes out).

Reply:
Ok I reformatted the table to be explicit about each option and supported version. Let me know if this makes more sense.
https://datadoghq.atlassian.net/wiki/spaces/TS/pages/3119775797/Troubleshooting+APM+Library+Configuration+via+the+UI+An+Overview#Current-Status-Public-Beta

Reply:
Got it--that makes sense to me now. I'll get that adjusted in the draft for us 👍

Reply:
@brett0000FF please take another look at the table in confluence, I had the Python support incorrect.